xio/decorator.go

```diff
@@ -55,10 +56,16 @@ func RateLimit(limit int) WriterDecorator {
 
 // SizeLimit decorator is used to add a size limit (number of bytes) for io.Writer.
 // If written bytes length exceeds the passed limit it will return an error.
-func SizeLimit(size int) WriterDecorator {
+func SizeLimit(size int, allow bool) WriterDecorator {
 	return func(writer io.Writer) io.Writer {
 		return WriterFunc(func(p []byte) (int, error) {
 			if len(p) > size {
```

Comment:
```suggestion
			if len(p) > size && !allow {
			    return 0, ErrSizeLimitExceeded
			}
```

Reply:
this logic is done by line 69

Reply:
Yes, but haw else can we know the message exceeded the limit?

Reply:
But this is counter (increased on this error) and triggers around its value are the only way to give a hint that app is sending to big entries.